### DISCUSSION TITLE
你们对同事下属，是不是也洗脑。老油条对不听控制不排华的，公报私仇给小鞋穿。要出国必须先树敌。

### DISCUSSION THREAD

**Discussion by transtorch** [The-Run-Philosophy-Organization/run#517] [General]
一名管理层是排华派，追到工位上骚扰偷窥员工。用git写代码之前发誓排华。把不听话的默默拉黑扣分

垃圾信息了属于是。没公德心。这几个contributor都挺傻逼。

**Comment by ghost** [comment]
怎么这个仓库最近出现了一堆新号 ...
老问题，请指出这个仓库的那个部分涉嫌排华？

**Reply by xiaoruyi2020** [reply]
那必须的啊，你不出去就只能他们自个儿被欺负，你出去了，他们就可以欺负你了。ahahah
